wsapi.go

```diff
@@ -328,7 +328,7 @@ type updateStatusOp struct {
 // If game!="" then set game.
 // If game!="" and url!="" then set the status type to streaming with the URL set.
 // if otherwise, set status to active, and no game.
-func (s *Session) UpdateStreamingStatus(idle int, game string, url string) (err error) {
+func (s *Session) UpdateStreamingStatus(idle int, game string, url string, isSong bool) (err error) {
```

Comment:
This will break existing users of the API, I think just the change to line 337 is enough to allow this.

If you want a simple way to update the status, I suggest just adding an 'UpdateListeningStatus' method, but I would shy away from that personally.

Reply:
Branch changed, apoligies. I think the ability to send the listening status should be added, however I'm not sure how you'd best want me to add this if not by adding an 'UpdateListeningStatus' method.

Reply:
UpdateStatusComplex would be the way to call this, adding some additional comments on the method could explain this to users.